src/config/db.js

```diff
@@ -13,6 +13,7 @@ const connectDB = async () => {
     });
     console.log('MongoDB Connected...');
   } catch (err) {
+    console.log('Mongo connection error')
```

Comment:
Please, add `err` as a second argument to the `console.log` to see the full error object